src/passwordValidation.js

```diff
@@ -10,13 +10,27 @@ export default function isValidPassword(password = "") {
   // The following line ensures, that password is always a string, like the number 128 -> string "128"
   if (typeof password !== "string") password = String(password);
   if (password.length !== 10) return false;
-  if (password.match(/[a - z][A - Z]/)) return true;
-  if (!password.match(/^[0-9a-zA-Z]+$/)) return false;
-  if (password.match(/\d+/)) return false;
-  if (forbiddenPasswords.includes(password)) {
+  //check if password are among in the forbbiden password
+  if (forbiddenPasswords.includes(password)) return false;
```

Comment:
for the best practice this should be the second test so that the following sequence of tests are to only check if that password in valid in form regex, it would be better if the 2nd test is this one and the third test is the one on set and checking unique characters and then the regexes continues